client/ratings_reviews/components/RatingBreakdown.jsx

```diff
@@ -1,9 +1,63 @@
 import React from 'react';
+import $ from 'jquery';
+import helpers from '../helpers.js';
 
-const RatingBreakdown = (props) => (
-  <div>
-    ratingbreakdown
-  </div>
-);
+class RatingBreakdown extends React.Component {
+  constructor(props) {
+    super(props);
+
+    $.ajax({
```

Comment:
not totally sure but it might be better to make the .ajax call in your top-level widget's component did mount, then pass the necessary data down via props. might not be better--not sure...

Reply:
@matthewwrobel Totally agreed! After today's discussion, I plan to refactor my AJAX calls to inside componentDidMount. Thank you for the reminder!